embedded-files/fuseml-core-deployment.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: fuseml-core
+  namespace: fuseml-core
+  labels:
+    app.kubernetes.io/name: fuseml-core
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: fuseml-core
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: fuseml-core
+    spec:
+      containers:
+      - name: fuseml-core
+        image: ghcr.io/fuseml/fuseml-core:latest
+        env:
+          - name: GITEA_USERNAME
+            valueFrom:
+              secretKeyRef:
+                key: GITEA_USERNAME
+                name: fuseml-core-gitea
+          - name: GITEA_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                key: GITEA_PASSWORD
+                name: fuseml-core-gitea
+          - name: GITEA_URL
+            valueFrom:
+              secretKeyRef:
+                key: GITEA_URL
+                name: fuseml-core-gitea
+        imagePullPolicy: IfNotPresent
```

Comment:
Combining the use of `latest` as an image tag with an `IfNotPresent` image pull policy is treacherous, because you'll find yourself using outdated images (see [https://kubernetes.io/docs/concepts/containers/images/#updating-images](updating-images)). I recommend removing the explicit image pull policy value.

Reply:
OK